SYCL/ESIMD/api/functional/ctors/operator_assignment.hpp

```diff
@@ -0,0 +1,97 @@
+//===-- operator_assignment.hpp - Functions for tests on simd assignment
+//      operators. --------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+///
+/// \file
+/// This file provides functios for tests on simd assignment operators.
+///
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+// The test proxy is used to verify the move constructor was called actually.
+#define __ESIMD_ENABLE_TEST_PROXY
+
+#include "../shared_element.hpp"
+#include "common.hpp"
+
+namespace esimd_test::api::functional::ctors {
```

Comment:
Wait a minute. Why do we have a tests for `operator` in `constructors` namespace and folder?

Reply:
I've done it in this commit: 4be7ac99eca7778991263036b9340b4580e5e378